source/guide_cryptpad.rst

```diff
@@ -108,28 +93,35 @@ Update configuration
 
 Open ``config/config.js`` in an editor and edit following lines:
 
-1. Uncomment the line beginning with ``//httpSafeOrigin:`` by removing the two slashes, and replace your instance URL like so:
+1. Uncomment the line beginning with ``//httpUnsafeOrigin:`` by removing the two slashes, and replace your instance URL like so:
```

Comment:
Why should the *un*safe origin be an HTTPS URL? I'm pretty sure this needs to stay as it is.